src/nunit.analyzers.tests/TestMethodUsage/TestMethodUsageAnalyzerTests.cs

```diff
@@ -51,6 +51,29 @@ public void VerifySupportedDiagnostics()
                 $"{TestMethodUsageAnalyzerConstants.SpecifiedExpectedResultForVoidMethodMessage} is missing.");
         }
 
+
+        private static IEnumerable<TestCaseData> SpecialConversions
```

Comment:
Note to reviewer: These have been duplicated across test files because I couldn't find a good place to centralize them. Please let me know if you have any suggestions, or think its ok as-is.

Reply:
I think it is fine as it is, but perhaps also add a testcase with a time and offset for `DateTimeOffset`.